File: examples/java/kora-java-crud/src/main/java/io/koraframework/example/crud/service/PetCircuitBreaker.java
package io.koraframework.example.crud.service;

import io.koraframework.resilient.circuitbreaker.CircuitBreaker;
import io.koraframework.resilient.circuitbreaker.annotation.CircuitBreakerSpec;

@CircuitBreakerSpec("resilient.circuitbreaker.pet")
public interface PetCircuitBreaker extends CircuitBreaker {

}

File: examples/kotlin/kora-kotlin-crud-submodule/kora-kotlin-crud-submodule-pet-api/src/main/kotlin/io/koraframework/kotlin/example/submodule/pet/service/PetCircuitBreaker.kt
package io.koraframework.kotlin.example.submodule.pet.service

import io.koraframework.resilient.circuitbreaker.CircuitBreaker
import io.koraframework.resilient.circuitbreaker.annotation.CircuitBreakerSpec

@CircuitBreakerSpec("resilient.circuitbreaker.pet")
interface PetCircuitBreaker : CircuitBreaker
